documentation/publish/distribute-pre-release-versions.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: sidebar
+title: Distribute pre-release versions
+permalink: /documentation/publish/distribute-pre-release-versions/
+published: true
+topic: Publish
+tags: [beta, alpha, testing, pre-release, publish, guide, how-to]
+contributors: [caitmuenster]
+last_updated_by: caitmuenster
+date: 2020-05-06
+---
+
+<!-- Page Hero Banner -->
+
+{% capture page_hero_banner_content %}
+
+# Distribute pre-release versions
+
+Learn how to distribute an alpha or beta pre-release version of your extension for Firefox.
+
+{% endcapture %}
+{% include modules/page-hero.html
+    content=page_hero_banner_content
+%}
+
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+Pre-release channels are not supported on [addons.mozilla.org](https://addons.mozilla.org) (AMO), so if you would like to have a limited group of users test a beta version of your extension, you will need to take the following steps to set up your own channel using an self-hosted version of your extension:
+
+1. In manifest.json of the beta version, [specify the location of your update manifest](/documentation/manage/updating-your-extension/#enable-update). This will ensure that your beta users will receive future updates. If your release channel is also self-hosted, you will need to use a different update URL for the beta channel.
```

Comment:
```suggestion
1. In `manifest.json` of the beta version, [specify the location of your update manifest](/documentation/manage/updating-your-extension/#enable-update). This will ensure that your beta users will receive future updates. If your release channel is also self-hosted, you will need to use a different update URL for the beta channel.
```